robottelo/api/utils.py

```diff
@@ -440,8 +430,8 @@ def configure_provisioning(org=None, loc=None, compute=False, os=None):
             .search(
                 query={
                     'search': 'family="Redhat" '
-                    'AND major="{0}" '
-                    'AND minor="{1}")'.format(
+                    'AND major="{}" '
+                    'AND minor="{}")'.format(
                         os.split(' ')[1].split('.')[0], os.split(' ')[1].split('.')[1]
                     )
                 }
```

Comment:
This looks pretty ugly still, and could still look better as an f-string. Perhaps something like this
```python
os_ver = os.split(' ')[1].split('.')
query={'search': 'family="Redhat" AND major="{os_ver[0]}" AND minor="{os_ver[1]}")'}
```